src/App.tsx

```diff
@@ -48,8 +54,11 @@ export const quoteCardTopColor = getRandomColor()
 export const quoteCardBottomColor = '#cccccc'
 //#endregion
 
-//#region Types 
-export type ToggleFilterFunction = (type:"author" | "tag", filter:string)=>void
+//#region Types
```

Comment:
@DeathstarNovember looks like your formatter did a lot of work here.

Reply:
Sure did!